dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionRestRepositoryIT.java

```diff
@@ -519,6 +519,108 @@ public void createTest() throws Exception {
+            .contentType(contentType))
+                            .andExpect(status().isCreated())
+                            .andExpect(content().contentType(contentType))
+                            .andExpect(jsonPath("$", Matchers.allOf(
+                                hasJsonPath("$.id", not(empty())),
+                                hasJsonPath("$.uuid", not(empty())),
+                                hasJsonPath("$.name", is("Title Text")),
+                                hasJsonPath("$.handle", not(empty())),
+                                hasJsonPath("$.type", is("collection")),
+                                hasJsonPath("$.metadata", Matchers.allOf(
+                                    MetadataMatcher.matchMetadata("dc.description",
+                                        "<p>Some cool HTML code here</p>"),
+                                    MetadataMatcher.matchMetadata("dc.description.abstract",
+                                        "Sample top-level community created via the REST API"),
+                                    MetadataMatcher.matchMetadata("dc.description.tableofcontents",
+                                        "<p>HTML News</p>"),
+                                    MetadataMatcher.matchMetadata("dc.rights",
+                                        "Custom Copyright Text"),
+                                    MetadataMatcher.matchMetadata("dc.title",
+                                        "Title Text")
+                                )))));
+
+    }
+
+    @Test
+    public void createTestByUnauthorizedUser() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and one collection.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .withLogo("ThisIsSomeDummyText")
+                                          .build();
+
+        ObjectMapper mapper = new ObjectMapper();
+        CollectionRest collectionRest = new CollectionRest();
+        // We send a name but the created collection should set this to the title
+        collectionRest.setName("Collection");
+
+        collectionRest.setMetadata(new MetadataRest()
+            .put("dc.description",
+                new MetadataValueRest("<p>Some cool HTML code here</p>"))
+            .put("dc.description.abstract",
+                new MetadataValueRest("Sample top-level community created via the REST API"))
+            .put("dc.description.tableofcontents",
+                new MetadataValueRest("<p>HTML News</p>"))
+            .put("dc.rights",
+                new MetadataValueRest("Custom Copyright Text"))
+            .put("dc.title",
+                new MetadataValueRest("Title Text")));
+
+        context.setCurrentUser(eperson);
```

Comment:
Again, after this line we should have a `context.restoreAuthSystemState();` to turn the authorization system back "on".

Reply:
@tdonohue ,  fixing these test problems in the next commit.

Looking things over, I notice quite a few tests to not restoreAuthSystemState, which surprised me.  They're not all simple GET requests, and some go to the trouble of passing a token.   So I'm a little confused about the effect of restoring authorization in the integration tests. It seems like it should always be done (even if not testing authorization) but it's not...

Reply:
@mspalti : Yes, restoring authorization should always be done. I try to look for it in reviews, but may not always catch it.  While in most scenarios the restoring of authorization may not be a big deal either way, it could turn out to be problematic to maintenance if later code is added that is affected by the lack of authorization checks -- i.e. it's very possible a test could be succeeding simply cause authorization checks are disabled, and *not* because it actually "works".

In any case, feel free to clean up others if you ever find them around.  I do try to check for it in all code reviews, but some may have slipped through the cracks.

Reply:
I updated the community and collection repository tests by restoring authorization where i found it missing.